.github/workflows/generate_markdown_files.yml

```diff
@@ -8,9 +8,6 @@ jobs:
     steps:    
       - name: "Checkout"
         uses: actions/checkout@v3
-        with:
```

Comment:
oh without these the bot won't auto run tests after committing files (if markdown files or 3rd party notices are added by github)